### PR TITLE
Update meta.yaml file for v2.6rc4

### PR DESCRIPTION
**Pull request by sloosvel**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
~* [ ] Bumped the build number (if the version is unchanged)~
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Review by valeriupredoi — APPROVED** [review]
looking good, cheers much @sloosvel

**Comment by zklaus** [comment]
This has not been rerendered yet. @valeriupredoi, please give the author a chance to finish the checklist before approving.

**Comment by zklaus** [comment]
How do you know that there is nothing to render?

**Comment by valeriupredoi** [comment]
> How do you know that there is nothing to render?

I thought Saskia only wanted to change the meta file, only to release rc4 after

**Comment by sloosvel** [comment]
I know it's not rendered please give a break I am on it!!

**Comment by sloosvel** [comment]
@conda-forge-admin, please rerender

**Comment by sloosvel** [comment]
Ok now, sorry I could not finish it yesterday

**Review by schlunma — APPROVED** [review]
Looking good, thanks Saskia! :rocket:

**Comment by sloosvel** [comment]
Thank you all!

**Comment by sloosvel** [comment]
(I can't merge this btw)

**Comment by schlunma** [comment]
Ah, sorry, we should really merge `main` into this at some point

**Comment by zklaus** [comment]
Nice work, thanks @sloosvel and everyone!

Just as an explanation: In many feedstocks the `meta.yaml` is the only file that is handled manually. The rerender produces most of the rest of the files, such as CI scripts. This means putting some information from the `meta.yaml` in other places, such as the list of maintainers in the README and in the github team, but also integrating any updates in the conda-forge system into the feedstock. See dc8922e9d3edc7480a9b5ac6519b72fa7262d129 for an example where the base container for the linux builds was changed. This is also important when [the global pinning](https://conda-forge.org/docs/maintainer/pinning_deps.html#globally-pinned-packages) changes.

tldr: The rerender depends on the state of conda-forge outside of this repo and it's basically impossible to know for sure if anything needs to be done or not without trying.
